src/CalcViewModel/StandardCalculatorViewModel.cpp

```diff
@@ -1558,29 +1534,15 @@ bool StandardCalculatorViewModel::IsRecoverableCommand(int nOpCode)
         return true;
     }
 
-    static Command recoverableCommands[] = { Command::CommandA, Command::CommandB, Command::CommandC, Command::CommandD, Command::CommandE, Command::CommandF };
+    static constexpr Command recoverableCommands[] = { Command::CommandA, Command::CommandB, Command::CommandC,
+                                                       Command::CommandD, Command::CommandE, Command::CommandF };
 
-    for (unsigned int i = 0; i < size(recoverableCommands); i++)
-    {
-        if (nOpCode == static_cast<int>(recoverableCommands[i]))
-        {
-            return true;
-        }
-    }
-    return false;
+    return find(begin(recoverableCommands), end(recoverableCommands), static_cast<Command>(nOpCode)) != end(recoverableCommands);
 }
 
 size_t StandardCalculatorViewModel::LengthWithoutPadding(wstring str)
 {
-    size_t count = 0;
-    for (size_t i = 0; i < str.length(); i++)
-    {
-        if (str[i] != L' ')
-        {
-            count++;
-        }
-    }
-    return count;
+    return std::count(str.begin(), str.end(), L' ');
```

Comment:
This code counts the number of space characters whereas the original code does the contrary. 

Suggestion:
`return str.length() - count(...)`

Reply:
You can remove `std::` too.

Reply:
Oh man, I can't believe I missed that `!=` in the original code.  I sure am glad you caught that.